src/multiplayer-servers/getting-started/cloud-provider-setup.md

```diff
@@ -53,3 +53,294 @@ In order to link your billing account to the projects managed by Nitrado, and to
+      "autoscaling:DeletePolicy",
+      "autoscaling:DeleteRole",
+      "autoscaling:DeleteRolePolicy",
+      "autoscaling:DetachRolePolicy",
+      "autoscaling:GetOpenIDConnectProvider",
+      "autoscaling:PutRolePolicy"
+    ]
+    resources = ["*"]
+  }
+}
+
+data "aws_iam_policy_document" "deployer_aux" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "kms:CreateAlias",
+      "kms:CreateGrant",
+      "kms:CreateKey",
+      "kms:DeleteAlias",
+      "kms:EnableKeyRotation",
+      "kms:ListAliases",
+      "kms:ScheduleKeyDeletion",
+      "kms:TagResource"
+    ]
+    resources = ["*"]
+  }
+  statement {
+    effect = "Allow"
+    actions = [
+      "iam:AttachRolePolicy",
+      "iam:CreateOpenIDConnectProvider",
+      "iam:CreatePolicy",
+      "iam:CreateRole",
+      "iam:DeleteOpenIDConnectProvider",
+      "iam:DeletePolicy",
+      "iam:DeleteRole",
+      "iam:DeleteRolePolicy",
+      "iam:DetachRolePolicy",
+      "iam:GetOpenIDConnectProvider",
+      "iam:GetPolicy",
+      "iam:GetPolicyVersion",
+      "iam:GetRole",
+      "iam:GetRolePolicy",
+      "iam:ListInstanceProfiles",
+      "iam:ListInstanceProfilesForRole",
+      "iam:ListPolicyVersions",
+      "iam:ListRolePolicies",
+      "iam:PutRolePolicy",
+      "iam:TagRole",
+      "iam:TagPolicy",
+      "iam:ListAttachedRolePolicies",
+      "iam:PassRole",
+      "iam:TagOpenIDConnectProvider",
+      "iam:ListOpenIDConnectProviders",
+    ]
+    resources = ["*"]
+  }
+  statement {
+    effect = "Allow"
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:DescribeLogGroups",
+      "logs:ListTagsForResource",
+      "logs:PutRetentionPolicy",
+      "logs:TagResource",
+      "logs:DeleteLogGroup",
+    ]
+    resources = ["*"]
+  }
+}
+
+# Merge all policy docs together
+data "aws_iam_policy_document" "combined_policy_documents" {
+  source_policy_documents = [
+    data.aws_iam_policy_document.deployer_eks.json,
+    data.aws_iam_policy_document.deployer_ec2.json,
+    data.aws_iam_policy_document.deployer_autoscaling.json,
+    data.aws_iam_policy_document.deployer_aux.json
+  ]
+}
+
+resource "aws_iam_role_policy" "gamefabric_operators" {
+  name   = "gamefabric-operators"
+  role   = aws_iam_role.gamefabric_operators.id
+  policy = data.aws_iam_policy_document.combined_policy_documents.json
+}
+```
+
+::: info Policy Document Size Restrictions
+AWS encodes their policy documents in JSON, and restricts the size of these documents rather strictly to 4KB per document.
+Adding all the permissions into one policy exceeds the limit for a single policy. We split them up logically such that
+they are grouped broadly by the resource types they grant access to.
+:::
+
+### Confirming the setup
+
+After you have completed the steps above, please contact Nitrado to confirm that the setup is complete, and provide us with the ARN of the role you created previously, as we need it on our end to access your account.
```

Comment:
Where can the ARN be found?

Reply:
The role's ARN is exported as part of the terraform run. If doing the setup manually, we should add another remark to "assume familiarity" with general AWS terms